client/src/components/Messages/Message.jsx

```diff
@@ -80,9 +80,9 @@ export default function Message({
     if (isJson(json)) {
       json = JSON.parse(json);
       if (json.code === 'invalid_api_key') {
-        return 'Invalid API key. Please check your API key and try again. You can access your API key by clicking on the model logo in the top-left corner of the textbox.';
+        return 'Invalid API key. Please check your API key and try again. You can do this by clicking on the model logo in the left corner of the textbox and selecting "Set Token" for the current selected endpoint. Thank you for your understanding.';
```

Comment:
This component is already bloated. If you could move this error handling out of the component into a utility function, that would help

Reply:
This was fixed in the latest commit.